model_compression_toolkit/exporter/model_exporter/keras/fakely_quant_tflite_exporter.py

```diff
@@ -18,6 +18,11 @@
 
 import keras.models
 import tensorflow as tf
+from packaging import version
+if version.parse(tf.__version__) >= version.parse("2.13"):
+    SAVE_FORMAT = ".keras"
+else:
+    SAVE_FORMAT = ".h5"
```

Comment:
I think it's better to define this once in the base keras exporter and use it in all other parts (instead of duplicating it).
Also, since it's used only for tmp file saving (like tflite export or tests), I would consider renaming it to something like "TMP_KERAS_EXPORT_FORMAT" so we will avoid confusion in the future (since the user can pass a path with either h5 or keras extension)